nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -422,15 +480,11 @@ in
           ListenAddress ${addr}${if port != null then ":" + toString port else ""}
         '') cfg.listenAddresses}
 
-        ${optionalString cfgc.setXAuthLocation ''
-            XAuthLocation ${pkgs.xorg.xauth}/bin/xauth
-        ''}
-
-        ${if cfg.forwardX11 then ''
-          X11Forwarding yes
-        '' else ''
-          X11Forwarding no
-        ''}
+        ${if cfg.forwardX11
```

Comment:
You are now just deleting newlines, which might mean the subsequent configuration is concatenated resulting in invalid configuration (didn't check, but that's why I believe it was done like that).

Reply:
Me tested to local VM - the configuration is generated normally

Reply:
OK, thanks for the confirmation.